src/service/video-manager-impl.js

```diff
@@ -393,6 +393,17 @@ export class VideoManager {
   isRollingAd(videoOrElement) {
     return this.getEntry_(videoOrElement).isRollingAd();
   }
+
+  /**
+   * @param {!VideoEntry} entryBeingPlayed
+   */
+  pauseOtherVideos(entryBeingPlayed) {
+    this.entries_.forEach((entry) => {
+      if (entry !== entryBeingPlayed && entry.isPlaying()) {
```

Comment:
Let's check for `PLAYING_MANUAL` here, otherwise we'd be pausing muted videos that may be used for presentation (like background videos).

Reply:
Done.

Reply:
Can we also get an exception for Stories? Thanks!

https://github.com/ampproject/amphtml/blob/b5238ef7bfcceb1cee5929c2fe209f0c4feaf747/extensions/amp-video/0.1/amp-video.js#L696-L702

Reply:
Happy to do this but for my learning, why isn't this desired in Stories?

Reply:
Thank you!
Stories have a lot of logic to handle video playback, coupled to page to page navigations or user interactions such as long presses. This feature you're introducing would potentially interact with the Stories logic in unexpected ways. For example, if a story has 3 videos on the same page we're expecting them to play simultaneously.

Reply:
It's fine to use the `-poolbound` classname for exclusion here.

Although the `VideoManager` already differentiates for Stories via the `AUTOPLAY_DELEGATED` signal, which we can also overload for this use case. (Maybe rename to `PLAYBACK_DELEGATED` to make it semantically appropriate?)

Tangentially, @gmajoulet: what does Stories use from the `VideoManager` currently? Analytics?

Reply:
Nothing that I know of actually. Do you know if video analytics work within a Story? I never tested it

Reply:
Updated the PR to exclude Stories, @gmajoulet can you confirm this change has no impact on Stories?